src/components/OSCALControlProse.js

```diff
@@ -102,6 +115,23 @@ function getConstraintsDisplay(modificationSetParameters, parameterId) {
   return constraintDescriptions.join("\n");
 }
 
+/**
+ * Retrieve the set-parameters when contained in the top-level implemented requirement statement
+ *
+ * @param {Object} implReqStatements statements on how the control is implemented
+ * @param {String} componentId uuid of the component and desired by-component
+ * @returns An array of set-parameters
+ */
+function getImplReqSetParameters(implReqStatements, componentId) {
+  // Get the top-level implemented requirement statement
+  const topLevelByComp = implReqStatements
+    ?.find((statement) => statement["statement-id"].endsWith("_smt"))
+    ?.["by-components"]?.find(
```

Comment:
```suggestion
    ?.["by-components"]
    ?.find(
```

Would the linter allow this? Makes the chain a little more "obvious"

Reply:
I've tried locally but whenever I run `npm run lint`, the suggested change is fixed to what it originally looked like so I don't think so.